packages/animated/src/AnimatedValue.ts

```diff
@@ -60,8 +61,9 @@ export class AnimatedValue<T = unknown> extends Animated
       this.startPosition = this.value
       this.lastPosition = this.value
       this.lastVelocity = isActive ? this.lastVelocity : undefined
-      this.lastTime = isActive ? this.lastTime : undefined
       this.startTime = G.now()
+      this.lastTime = isActive ? this.lastTime : this.startTime
+      this.elapsedTime = isActive ? this.elapsedTime : 0
```

Comment:
Can `decay` and `easing` animations actually continue seamlessly from a previous update? I doubt it, but I haven't tried. If not, let me know and I'll push a local commit I have that changes this.

Reply:
I haven’t touched this so I suspect they don’t. `elapsedTime` is just a timer that accounts for lost frames for `decay` and `easing` :)

Reply:
I’m on my phone but the commit looks perfect, I guess `progress` only makes sense for `easing`?